=== modified file 'bin/osv/orm.py'
--- bin/osv/orm.py	2011-05-24 10:18:10 +0000
+++ bin/osv/orm.py	2011-06-07 20:17:29 +0000
@@ -2449,6 +2450,9 @@
                 if k.startswith('x_') and not update_custom_fields:
                     continue
 
+                if k.startswith('x_js_'):
+                    continue
+                
                 f = self._columns[k]
 
                 if isinstance(f, fields.one2many):

